### PR TITLE
Readthedocs removed & students asking questions & python judge nl

### PR DESCRIPTION
**Pull request by BTWS2**

### CODE REVIEW

**Review by bmesuere — CHANGES_REQUESTED** [review]
- I'm not sure I like the navbar icon.
- The old docs link can of course be removed, but the homepage becomes a bit light on text. Maybe "manuals" and "reference pages" in the first paragraph can be linked to their respective pages. A second paragraph linking to the getting started for teachers would also be welcome.

**Comment by BTWS2** [comment]
Adding a new paragraph moves the "manuals" and "reference" buttons down, so I added the links. If there isn't a lot of text the buttons for it are just beneath it:

image

After changes:
image

Do I have to remove the logo?

**Comment by BTWS2** [comment]
Commit *students asking questions* is an attempt at resolving #98.

**Comment by BTWS2** [comment]
@bmesuere do you have time in the coming weeks to review this pull request?

File: en/README.md [hunk 1]
```diff
@@ -7,9 +7,9 @@ actionLink: "/nl/"
 footer: Made by Team Dodona with ❤️
 ---
 
-On this website, you can find a number of manuals and reference pages. The ultimate goal of this web site is to offer all content in both English and Dutch. For the time being, not all pages are translated.
+On this website, you can find a number of [guides](/guides) and [reference pages](/references). The ultimate goal of this web site is to offer all content in both English and Dutch. For the time being, not all pages are translated.
 
-This website will become the definitive source of all Dodona documentation. While we migrate the content, some information can still be found at our old site at [https://dodona.readthedocs.io/nl/latest/](https://dodona.readthedocs.io/nl/latest/).
+Are you a teacher and do you want to create your own Dodona course (with existing or own programming exercises)? Have a look at [this getting started guide](guides/teachers/getting-started).
```
Comment (bmesuere) [inline en/README.md:12]:
```suggestion
If you are a teacher and want to create your own Dodona course (with existing or own programming exercises), have a look at [this getting started guide](guides/teachers/getting-started).
```

File: nl/guides/students/exercises/README.md [hunk 2]
```diff
@@ -144,3 +144,22 @@ Als de judge binnen een testgeval rapporteert over individuele testen, dan worde
     ![image](./student.exercise_feedback_incorrect_tab.png)
 
     ![image](./student.exercise_feedback_visual.nl.png)
+
+## Vragen stellen
+![image](./student.ask_questions.png)
+Nadat je je oplossing ingediend hebt, kan je op twee manieren een vraag stellen. Bovenaan de ingediende code kan je een algemene vraag stellen door op `STEL EEN VRAAG OVER JE CODE` te klikken. Daarnaast kan je door links van het regelnummer op de oranje cirkel te klikken een vraag stellen bij een specifieke regel code.
```
Comment (bmesuere) [inline nl/guides/students/exercises/README.md:150]:
```suggestion
Nadat je je oplossing hebt ingediend, kan je op twee manieren een vraag stellen. Bovenaan de ingediende code kan je een algemene vraag stellen door op `Stel een vraag over je code` te klikken. Daarnaast kan je links van het regelnummer op de oranje cirkel klikken een vraag stellen bij een specifieke regel code.
```

File: nl/guides/students/exercises/README.md [hunk 3]
```diff
@@ -144,3 +144,22 @@ Als de judge binnen een testgeval rapporteert over individuele testen, dan worde
...
+
+![image](./student.type_questions.png)
+
+Typ in het tekstvak de vraag die je aan je leerkracht wil stellen. Je kan Markdown gebruiken om je tekst extra opmaak te geven. Klik als laatste op `VRAAG STELLEN`.
```
Comment (bmesuere) [inline nl/guides/students/exercises/README.md:154]:
```suggestion
Typ in het tekstvak de vraag die je aan de lesgever wil stellen. Je kan Markdown gebruiken om je tekst extra opmaak te geven. Klik als laatste op `Vraag stellen`.
```

File: nl/guides/students/exercises/README.md [hunk 4]
```diff
@@ -144,3 +144,22 @@ Als de judge binnen een testgeval rapporteert over individuele testen, dan worde
...
+
+::: tip Ondersteuning voor Markdown
+
+Je kan een aantal woorden in een vraag beklemtonen door:
```
Comment (bmesuere) [inline nl/guides/students/exercises/README.md:158]:
```suggestion
Je kan met Markdown extra opmaak toevoegen door:
```

File: nl/guides/students/exercises/README.md [hunk 5]
```diff
@@ -144,3 +144,22 @@ Als de judge binnen een testgeval rapporteert over individuele testen, dan worde
...
+
+- backticks (\` kan je typen door de toetscombinatie `Alt Gr`+ `µ`) rond een stukje code te zetten. \`Variabelen\` wordt bijvoorbeeld weergegeven als `Variabelen`.
+- asterisken (\*) of liggende streepjes (\_) rond woorden te zetten om het schuin weer te geven. \*schuine tekst\* wordt bijvoorbeeld weergegeven als *schuine tekst*.
+- twee asterisken (\**) of twee liggende streepjes (\__) rond woorden te zetten om het in het vet weer te geven. \*\*vette tekst\*\* wordt bijvoorbeeld weergegeven als **vette tekst**.
```
Comment (bmesuere) [inline nl/guides/students/exercises/README.md:162]:
```suggestion
- asterisken (\*)  rond woorden te zetten om het schuin weer te geven. \*schuine tekst\* wordt bijvoorbeeld weergegeven als *schuine tekst*.
- twee asterisken (\**) rond woorden te zetten om het in het vet weer te geven. \*\*vette tekst\*\* wordt bijvoorbeeld weergegeven als **vette tekst**.
- backticks (\`) rond een stukje code te zetten. \`Variabelen\` wordt bijvoorbeeld weergegeven als `Variabelen`.
```

**Review by bmesuere — APPROVED** [review]

**Comment by bmesuere** [comment]
Sorry for the long delay!

**Pull request merged**